src/client/api/project.js

```diff
@@ -1,6 +1,16 @@
 import EventEmitter from "eventemitter3";
 
-export default class Project extends EventEmitter {
+const projectDirectoryPath = "/api/files/";
+
+export function getRelativeURI(uri) {
```

Comment:
Because we are eventually going to want to make another project adapter for a hosted version of the editor, I'd rather these be instance methods on the Project class and pass the project class to the necessary components via the context API.